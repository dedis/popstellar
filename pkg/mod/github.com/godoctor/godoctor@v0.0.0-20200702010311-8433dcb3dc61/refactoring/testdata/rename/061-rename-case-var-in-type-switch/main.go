package main

import "fmt"


// Test for renaming the type switch variable
func main() {

var t interface{}
t = bool(true);           
switch y := t.(type) {           
default:
    fmt.Printf("unexpected type %T", y)  // <<<<< rename,13,38,13,38,renamed,pass   
case bool:
    fmt.Printf("boolean %t\n", y)            
case int:
    fmt.Printf("integer %d\n", y)            
case *bool:
    fmt.Printf("pointer to boolean %t\n", *y) 
case *int:
    fmt.Printf("pointer to integer %d\n", *y) 
}


 
}
