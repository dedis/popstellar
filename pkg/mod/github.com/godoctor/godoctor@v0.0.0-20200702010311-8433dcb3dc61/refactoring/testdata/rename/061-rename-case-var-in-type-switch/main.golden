package main

import "fmt"


// Test for renaming the type switch variable
func main() {

var t interface{}
t = bool(true);           
switch renamed := t.(type) {           
default:
    fmt.Printf("unexpected type %T", renamed)  // <<<<< rename,13,38,13,38,renamed,pass   
case bool:
    fmt.Printf("boolean %t\n", renamed)            
case int:
    fmt.Printf("integer %d\n", renamed)            
case *bool:
    fmt.Printf("pointer to boolean %t\n", *renamed) 
case *int:
    fmt.Printf("pointer to integer %d\n", *renamed) 
}


 
}
